src/color.scss

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2023 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Import Auro tokens
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+:host([visible]) {
+  background-color: var(--ds-auro-toast-background-color);
+  color: var(--ds-auro-toast-text-color);
+
+  .icon {
+    color: var(--ds-auro-toast-icon-color);
+  }
+
+  .closeButton {
+    background-color: var(--ds-auro-toast-close-button-background-color);
+    color: var(--ds-auro-toast-close-button-icon-color);
+
+    &:hover {
+      border: 1px solid var(--ds-auro-toast-close-button-border-color);
```

Comment:
This is a small pre-existing UI bug - the button bounces when you hover over it because the border isn't added until :hover.

1. Put the border color, style and size directly on .closeButton.
1. Make --ds-auro-toast-close-button-border-color default to `transparent`.
1. Update the button border color on :hover

Reply:
Proposed solution has been tested and does not work. We need to investigate another solution or a ticket should get made.

Reply:
I'd recommend creating a new issue to address this specifically. I attempted to update this with `outline` and that won't work because this is a color transparency, so you don't even see the outline.

Reply:
> Proposed solution has been tested and does not work. We need to investigate another solution or a ticket should get made.

The proposed solution works exactly as expected. See this commit: https://github.com/AlaskaAirlines/auro-toast/pull/38/commits/5a55ed97bbc0aeb33def33aa9625e9861ed816be

Issue is fixed and I am resolving the comment thread.